orchestrator/api/watches.py

```
@@ -0,0 +1,152 @@
+    status: Optional[str] = Query(
+        None, description="Filter by exact watch status (e.g. 'watching')"
+    ),
+    watch_type: Optional[str] = Query(
+        None, description="Filter by watch type (mission / playbook_execution / scheduled_playbook)"
+    ),
+    include_closed: bool = Query(
+        False, description="Without a status filter, include closed watches too"
+    ),
+    limit: int = Query(100, ge=1, le=200),
+    offset: int = Query(0, ge=0),
+    ctx: RequestContext = Depends(get_request_context_hybrid),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)